test-one/src/index.js

```diff
@@ -2,13 +2,21 @@ import { log } from "./helpers/logger";
 
 export const go = async () => {
   // Your code should go here instead of the welcome message below. Create and modify additional files as necessary/appropriate.
-
-  const welcomeMessage =
-    "Congrats, you're up and running! Welcome to the first moshtix code challenge! " +
-    "Please replace this message with your solution and feel free to add " +
-    "or modify other files as necessary. Good luck : )";
-
-  await log(welcomeMessage);
+  
+   
+    for(var i=1;i<=100;i++){
+      var output = "";    //generate a empty var
+      if(i%3==0){
+        output += "Boss"; //add Boss to output if true
+      }
+      if(i%5==0){
```

Comment:
👍 I like this idea of concatenating rather than having a separate condition for 'BossHog'